MAP/public/data/environment.response.json

```diff
@@ -31,7 +31,15 @@
             {
                 "name":"CountriesSearcher",
                 "is_visible": true
-            } 
+            },
+            {
+                "name":"Totals",
```

Comment:
since you chose this PR to make the name change, this should also be StatsBox (still a few other places too)